File: flink_demo/src/main/java/cpm/hdu/requestlog/RequestLogApplication.java
package cpm.hdu.requestlog;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

/**
 * @ClassName：RequestLogApplication
 * @Description：TODO
 * @Author：ssf
 * @Date：2021/3/18 11:30 上午
 * @Versiion：1.0
 */

public class RequestLogApplication {
    private static Logger logger = LoggerFactory.getLogger(RequestLogApplication.class);
    public static void main(String[] args){
        logger.info("begin stream handle");
//    PrintJob.start()
//    KafkaConsumerJob.start
//        KafkaConsumerTableJob.start();
        logger.info("end stream handle\"");
    }
}

File: flink_demo/src/main/scala/com/hdu/requestlog/RequestLogApplication.scala
package com.hdu.requestlog

import com.hdu.chapter14_flink.datastream.PrintJob
import com.hdu.requestlog.job.{KafkaConsumerJob, KafkaConsumerTableJob, RequestLogJob}
import org.slf4j.LoggerFactory


/**
 * @author shushoufu
 * @date 2020/08/26
 **/
object RequestLogApplication {
  final val logger = LoggerFactory.getLogger(this.getClass)

  @throws[Exception]
  def main(args: Array[String]): Unit = {
    logger.info("begin stream handle");
    //    PrintJob.start()
    //    KafkaConsumerJob.start
    KafkaConsumerTableJob.start()
    logger.info("end stream handle\"")
  }
}
